pkg/lwip/contrib/lwip.c

```diff
@@ -73,7 +75,8 @@ void lwip_bootstrap(void)
 #ifdef MODULE_NETDEV_TAP
     for (unsigned i = 0; i < LWIP_NETIF_NUMOF; i++) {
         netdev_tap_setup(&netdev_taps[i], &netdev_tap_params[i]);
-        if (netif_add(&netif[i], &netdev_taps[i], lwip_netdev_init,
+        netdev_eth_add((netdev_t*)&netdev_taps[i], &eth_layer[i]);
+        if (netif_add(&netif[i], &eth_layer[i], lwip_netdev_init,
```

Comment:
Ah... easier than I anticipated ^^

Reply:
Yeah, but not a solution that scales nicely. If we ever start supporting more devices we might want to refactor this to something that scales.

Reply:
Yupp, but then `lwip_netdev_init` needs some heavy refactoring anyways.